src/lib/orionld/serviceRoutines/orionldPostEntityOperationsUpsert.cpp

```diff
@@ -274,9 +277,7 @@ bool orionldPostEntityOperationsUpsert(ConnectionInfo* ciP)
         entityIdNodeP = itemP;
         next = itemP->next;
         kjChildRemove(entityNodeP, entityIdNodeP);
-
-        if (entityTypeNodeP != NULL)  // Both ID and type have been found - we are done
-          break;
```

Comment:
Good!
In order to make sure there are no duplicates, we have to check the entire object. Can't break when we've found some

Reply:
NTC (Nothing To Change)